src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --nav-font-size: 12px;
+  --action-color: #00acdc;
+}
+
+html,
+body {
+  font-family: "Roboto", sans-serif;
+}
```

```suggestion
html {
  font-family: "Roboto", sans-serif;
}
```
add fallback font
move font size here, you need to set it only once